image/image.go

```diff
@@ -263,3 +263,13 @@ func NewImgJSON(src []byte) (*Image, error) {
 	}
 	return ret, nil
 }
+
+// Check wheather id is a valid image ID or not
+func ValidateID(id string) error {
```

Comment:
hm, yes. `stringid.TruncateID()` is quite docker-specific though, because of the arbitrary length. Think that put me on the wrong foot.